tests/aux.cljs

```diff
@@ -46,3 +46,20 @@
 (defn sync-setup [f]
   (f)
   (tlr/cleanup))
+
+(defn screen [] (tlr/within js/document))
```

Comment:
I was using this for the "screen" function in another project
```clj
(defn screen []
  (tlr/getQueriesForElement (.-body js/document) tlr/queries))
```
This is much more elegant, thanks!